website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -78,22 +88,45 @@ Options:
       --network-file <path>     Networks config file path (default: "./networks.json")
 ```
 
-The `add` command will fetch the ABI from Etherscan (unless an ABI path is specified with the `--abi` option), and will create a new `dataSource` in the same way that `graph init` command creates a `dataSource` `--from-contract`, updating the schema and mappings accordingly.
+#### Specifics
+
+The `add` command will fetch the ABI from Etherscan (unless an ABI path is specified with the `--abi` option) and creates a new `dataSource`, similar to how the `graph init` command creates a `dataSource` `--from-contract`, updating the schema and mappings accordingly. This allows you to index implementation contracts from their proxy contracts.
+
+- The `--merge-entities` option identifies how the developer would like to handle `entity` and `event` name conflicts:
+
+  - If `true`: the new `dataSource` should use existing `eventHandlers` & `entities`.
+
+  - If `false`: a new `entity` & `event` handler should be created with `${dataSourceName}{EventName}`.
```

Comment:
Is the difference in pluralization between this line and the previous one intentional?
- `entities` vs `entity`
- `event` handler vs `eventHandlers` (also note that "handler" is not part of the inline code in the second line)

Reply:
@benface I assume that it is since it was not noted as an issue